server/data/licenceApiClient.ts

```diff
@@ -703,6 +704,12 @@ export default class LicenceApiClient extends RestClient {
     })) as Promise<boolean>
   }
 
+  async getLastMinuteCases(): Promise<LastMinuteHandoverCaseResponse[]> {
+    return (await this.get({
+      path: `/offender/support/report/last-minute-handover-cases`,
+    })) as Promise<LastMinuteHandoverCaseResponse[]>
+  }
```

Comment:
suggestion: should probably have a test for this

Reply:
have not done tests yet... just thought I could through it together and do a demo asap...

Reply:
we don't really have tests for most of the support routes and pages but do for everything else